### PR TITLE
Updated documentation for SNAPS-kubernetes

### PR DESCRIPTION
**Pull request by shreya218**
#### What does this PR do?
Updates the documentation for ci/README.md and doc/source/install/install.md.
Adds ci/Conformance.md for instructions on running Sonobuoy test and submitting the conformance results.
#### Do you have any concerns with this PR?
-
#### How can the reviewer verify this PR?
Execute the steps by following the instructions.
#### Any background context you want to provide?
Gives more detailed steps and notes on issues that might come across during the execution of test.
#### Screenshots or logs (if appropriate)
-
#### Questions:
- Have you connected this PR to the issue it resolves?
#issue 210
- Does the documentation need an update?
That is the purpose.
- Does this add new Python dependencies?
No
- Have you added unit or functional tests for this PR?
No
- Does this patch update any configuration files?
No

### CODE REVIEW

File: ci/Conformance.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+## Recommended host machine configuration
+1. RAM   : 32 GB
+2. vCPUs : 8  
+
+## Heptio Sonobuoy test
+The standard tool for running the conformance tests, Sonobuoy, is already installed while deploying the kubernetes cluster using snaps-orchestration.
+
+### Install Sonobuoy
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Put the install latest release section first and list it is (Recommend).

For the install latest say (Unstable) and include installing the latest version of go.

Don't use numbers for none sequential items.  Like install latest or top of tree.

File: ci/Conformance.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Say something like install sonobouy and have the configuration file in place.  Then validate with kubectl.

File: ci/Conformance.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Conformance results
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Provide a summary of the steps needed and the purpose of the conformance test.

Creating a flow diagram for extra credit.

File: ci/Conformance.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## To check for the existing nodes
+```
+$ kubectl get nodes
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Don't use the $ sign in the text.  This makes it more difficult to copy and past the commands.

File: ci/Conformance.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
+## To check for the existing nodes
+```
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Say bash as the language for the code block.

File: ci/Conformance.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+$ ./sonobuoy retrieve
+$ tar -xvzf {tarball}
+```
+Note: Make sure to give enough time for the tests to complete successfully; Trying to retrieve the result too soon will generate result with missing log files.
+### Delete pods and namespaces
+```
+$ ./sonobuoy delete --all
+```
+## Submit Conformance results
```
Comment (RandyLevensalor) [inline ci/Conformance.md]: Include a link to our latest results and say that the README.md and PRODUCT.yaml can be copied and updated as needed. https://github.com/cablelabs/k8s-conformance/tree/master/v1.14/snaps-kubernetes

Reference our fork for staging the changes. https://github.com/cablelabs/k8s-conformance

File: ci/README.md [hunk 7]
```diff
@@ -44,13 +44,14 @@ document.
 
 ## Setup & Execution
 
-### Download and install snaps-oo
+### Download and install snaps-orchestration
 
 ```bash
```
Comment (RandyLevensalor) [inline ci/README.md]: Use this as an example for the steps in Conformance.md

File: ci/README.md [hunk 8]
```diff
@@ -44,13 +44,14 @@ document.
...
-git clone https://github.com/opnfv/snaps.git
-sudo apt update
-sudo apt install python git python2.7-dev libssl-dev python-pip
-sudo pip install -e snaps
+git clone https://github.com/cablelabs/snaps-orchestration.git
+pip uninstall -y snaps-orchestration
+pip freeze | xargs pip uninstall -y
```
Comment (spisarski) [inline ci/README.md]: I don't know if you want to make everyone uninstall all of their python packages here...

File: ci/Conformance.md (hunk 4 above)
Comment (spisarski) [inline ci/Conformance.md]: Why?

File: ci/Conformance.md (hunk 1 above)
Comment (spisarski) [inline ci/Conformance.md]: need to install go here

File: ci/Conformance.md (hunk 2 above)
Comment (spisarski) [inline ci/Conformance.md]: On which server? I think we should only do on the build server...

File: ci/Conformance.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+## Heptio Sonobuoy test
+The standard tool for running the conformance tests, Sonobuoy, is already installed while deploying the kubernetes cluster using snaps-orchestration.
+
+### Install Sonobuoy
+1. For the latest version of CLI:
+```
+$ go get -u -v github.com/heptio/sonobuoy
+```
+2. For a specific version x of Sonobuoy:
+```
```
Comment (spisarski) [inline ci/Conformance.md]: May want to make note of the current version and why

File: ci/Conformance.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+$ wget {https://github.com/heptio/sonobuoy/releases/download/{version}/{version x tarball}}
+$ tar -xvzf {version x tarball}
+```
+### Run Sonobuoy
+```
+$ ./sonobuoy run --mode={optional flag quick or wait}
```
Comment (spisarski) [inline ci/Conformance.md]: note --kubeconfig=... parameter for all sonobuoy commands

File: ci/Conformance.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+```
+$ ./sonobuoy status
+```
+### Inspect the logs
+```
+$ ./sonobuoy logs
+```
+### View the status of the running sonobuoy pods
+```
+$ kubectl -n heptio-sonobuoy get all
```
Comment (spisarski) [inline ci/Conformance.md]: Remove kubectl in here and do everything via the sonobuoy client

File: ci/Conformance.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+```
+The logs will display the following message on successful completion of the Sonobuoy test: 
+```
+msg="no-exit was specified, sonobuoy is now blocking"
+``` 
+### Retrieve the results 
+```
+$ ./sonobuoy retrieve
+$ tar -xvzf {tarball}
+```
```
Comment (spisarski) [inline ci/Conformance.md]: Make note of the location of the e2e.log and junit.xml files necessary for CNCF are located

File: ci/README.md [hunk 13]
```diff
@@ -44,13 +44,14 @@ document.
...
-### Download and install snaps-oo
+### Download and install snaps-orchestration
 
 ```bash
-git clone https://github.com/opnfv/snaps.git
-sudo apt update
-sudo apt install python git python2.7-dev libssl-dev python-pip
-sudo pip install -e snaps
+git clone https://github.com/cablelabs/snaps-orchestration.git
+pip uninstall -y snaps-orchestration
```
Comment (spisarski) [inline ci/README.md]: No need to uninstall

File: ci/README.md [hunk 14]
```diff
@@ -88,15 +89,15 @@ short explanation of each variable:
 note: The entire process can take up to an hour to complete.
 
 ```bash
-python { path_to_snaps-oo}/examples/launch.py -t { path_to_snaps-kubernetes}/ci/snaps/snaps_k8_tmplt.yaml \
--e { path_to_env_file } -d
+python {repo_dir}/snaps-orchestration/openstack-launch.py -t {path to snaps template}/snaps_k8_tmplt.yaml -e {path to optional environment file for J2} -d
 ```
-
+Optional flag: -v {variable_name}={value} will override the default values in the environment file.
```
Comment (spisarski) [inline ci/README.md]: and add

File: doc/source/install/install.md [hunk 15]
```diff
@@ -64,11 +64,9 @@ components.
 
 | Category | Software version |
 | -------- | ---------------- |
-| Operating System | Ubuntu 16. |
-| Programming Language | Python 2.7.12 |
-| Automation | > Ansible 2.4 |
+| Operating System | Ubuntu 16. and Ubuntu 18. |
```
Comment (spisarski) [inline doc/source/install/install.md]: get rid of dots

File: doc/source/install/install.md [hunk 16]
```diff
@@ -64,11 +64,9 @@ components.
...
+| Programming Language | Python 2.7.x |
 | Framework |  Kubernetes v1.14.3 |
```
Comment (spisarski) [inline doc/source/install/install.md]: put in the supported range

File: doc/source/install/install.md [hunk 17]
```diff
@@ -88,15 +86,24 @@ baremetal host, it is recommended but not required to leverage SNAPS-Boot.
...
-    1. Node setup - Install prerequisites (i.e. docker-ce 17.03)
-    1. Kubernetes cluster deployment via Kubespray
-    1. Post installation processes such as CNI, node labeling, and metrics server installation
-
+1. Provision nodes with Ubuntu and configure network (see snaps-boot <https://github.com/cablelabs/snaps-boot>)
+1. Build server setup (SNAPS-Kubernetes)
+    1. Install prerequisites 
+        1. Software requirements
+            1. Node requirements : SSH keyed to cluster nodes
+            1. Python
```
Comment (spisarski) [inline doc/source/install/install.md]: apt install ...

File: ci/Conformance.md (hunk 4 above)
Comment (RandyLevensalor) [reply ci/Conformance.md]: Have you tried to copy and paste when you have to delete the first 2 characters of each line.  Without the leading '$' you can just double click and copy the entire text box.  Be it one line or 10.

File: doc/source/install/install.md (hunk 15 above)
Comment (RandyLevensalor) [reply doc/source/install/install.md]: I'd say Ubuntu 16.04 and 18.04.  We don't support other releases of Ubuntu 16 and 18.

File: ci/README.md [hunk 18]
```diff
@@ -44,13 +44,14 @@ document.
...
 ```bash
-git clone https://github.com/opnfv/snaps.git
-sudo apt update
-sudo apt install python git python2.7-dev libssl-dev python-pip
-sudo pip install -e snaps
+git clone https://github.com/cablelabs/snaps-orchestration.git
+pip uninstall -y snaps-orchestration
+pip freeze | xargs pip uninstall -y
+sudo pip install -r snaps-orchestration/requirements-git.txt
+sudo pip install -e snaps-orchestration/
```
Comment (RandyLevensalor) [inline ci/README.md]: @spisarski Can you make this a single pip install for all of the dependencies?

**Comment by shreya218** [comment]
Reviews integrated. Please confirm.

**Pull request merged**